src/lib/components/PromptWithSlider.svelte

```diff
@@ -94,7 +94,6 @@
 		display: flex;
 		align-items: center;
 		padding: 10px 16px;
-		border: 4px solid var(--cream);
```

Comment:
tiny change to prevent this from overlapping with the bg image visuals on smaller screen sizes